h2o-extensions/target-encoder/src/main/java/ai/h2o/targetencoding/TargetEncoder.java

```diff
@@ -64,9 +64,9 @@ public void init(boolean expensive) {
     }
   }
 
-  private void disableIgnoreConstColsFeature() {
+  private void disableIgnoreConstColsFeature(boolean expensive) {
     _parms._ignore_const_cols = false;
-    if (logger.isInfoEnabled())
+    if (expensive && logger.isInfoEnabled())
```

Comment:
non-related: disabled the warning at startup when building the builder prototypes